### DISCUSSION TITLE
2021年美国侵犯人权报告

### DISCUSSION THREAD

**Discussion by lxn7022** [The-Run-Philosophy-Organization/run#420] [General]
[https://m.gmw.cn/baijia/2022-03/01/35553853.html](url)
![image](https://user-images.githubusercontent.com/66630801/167532753-0925560c-daf2-4655-84d6-4b9d69637349.png)
![image](https://user-images.githubusercontent.com/66630801/167532776-538e9fb7-875e-4759-a9b2-9c699c8e845c.png)
![image](https://user-images.githubusercontent.com/66630801/167532786-d72312c6-1426-43aa-b6a7-a6ee5738edb5.png)
![image](https://user-images.githubusercontent.com/66630801/167532799-c61fcb91-694e-46a7-9b90-f0df2f6f5bbe.png)
![image](https://user-images.githubusercontent.com/66630801/167532813-b46b5aa2-15b6-4851-89d3-ad92d3d518fd.png)
![image](https://user-images.githubusercontent.com/66630801/167532822-ee4564e6-03f5-49fe-81b9-d1b851a97b3b.png)

**Comment by woqi** [comment]
你是脑残么？总在这个仓库发issue？到你自己的阵营当粉红去好么？

**Reply by lxn7022** [reply]
> 你是脑残么？总在这个仓库发issue？到你自己的阵营当粉红去好么？

这里有5大阵营：
https://www.toutiao.com/article/7102803353993789956
![image](https://user-images.githubusercontent.com/66630801/170918471-89ac2d54-65b5-4147-b013-3becceefd9d1.png)

**Comment by tonylyf407** [comment]
言论自由，发，都可以发

**Comment by ytgui** [comment]
说的好，省流总结：
+ 漠视人民生命权，孕妇在医院门口流产
+ 心理健康恶化，不少人民跳楼
+ 人民无家可归，甚至只能住在电话亭
+ 老年人生活困难，尤其是不能熟练使用手机的
+ 黑心疫苗公司垄断，至今没有明确的不良反应统计
+ 暴力执法将老人压在桌子下.jpg
+ 外国投资者被无法无纪震惊，大批离开
+ 严重通货膨胀，居民无法满足正常吃饭和买房需求

美国真是万恶不赦，搬起石头砸自己的脚，迟早完蛋

**Reply by lxn7022** [reply]
> 说的好，省流总结：
> 
> * 漠视人民生命权，孕妇在医院门口流产
> * 心理健康恶化，不少人民跳楼
> * 人民无家可归，甚至只能住在电话亭
> * 老年人生活困难，尤其是不能熟练使用手机的
> * 黑心疫苗公司垄断，至今没有明确的不良反应统计
> * 暴力执法将老人压在桌子下.jpg
> * 外国投资者被无法无纪震惊，大批离开
> * 严重通货膨胀，居民无法满足正常吃饭和买房需求
> 
> 美国真是万恶不赦，搬起石头砸自己的脚，迟早完蛋

![image](https://user-images.githubusercontent.com/66630801/170918576-782cf993-c386-48da-afaa-484b032389af.png)

**Comment by PeterWrighten** [comment]
我觉得这里很多issue可以汇集起来，出一个国人赢麻集锦，真的，这么长时间了，素材积累也够了。@the-run-philosophy

**Comment by MouxiaoHuang** [comment]
The United States is undoubtedly lifting a stone and hitting itself in the foot.😠

**Comment by shikelong** [comment]
说得好。美国石头都不够用了

**Comment by BlueWhale235** [comment]
为什么会有 **黄石公园** 呢，因为 **黄磊公园** 石头都被搬了三分之二了

**Comment by freestyletime** [comment]
> 言论自由，发，都可以发


言论没有绝对自由，不要让一个撒谎的人利用自由世界的规则来破坏自由。

**Comment by BrotaOne** [comment]
> 

如果你认为他在说谎，为什么不拿出证据？禁止别人发言，和你讨厌的人有什么不同？

**Comment by flin-ux** [comment]
常识问题需要证据吗？别人的时间精力不要钱吗？
你会去浪费时间证明饭香屎臭吗？

**Comment by jingyuexing** [comment]
> 你是脑残么？总在这个仓库发issue？到你自己的阵营当粉红去好么？

要让不同阵营的人说话 兼听则明 除非是自己恐惧 害怕

**Comment by jingyuexing** [comment]
> 我建议您开一个We-are-winer的库，告诉人们我们赢麻了

很难不支持

**Comment by PeterWrighten** [comment]
> 我建议您开一个We-are-winer的库，告诉人们我们赢麻了

附议

**Comment by trillionstar07** [comment]
低级的自由是想做什么就能做什么，高级的自由是不想做什么就可以不做什么

**Comment by Evcklli** [comment]
自己事情做不好整天挑别人毛病，这样搞比得过美国么？哈哈，看来贵支也知道自己比不过也就一边送二代过去一并给奴隶洗脑
